src/Bicep.MSBuild.E2eTests/examples/bicepconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    // See https://aka.ms/bicep/config for more information on Bicep configuration options
```

Comment:
Why was this added? I don't see any tests in the project

Reply:
They're in TypeScript, e.g.: bicep\src\Bicep.MSBuild.E2eTests\src\fail.test.ts